src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -1576,8 +1576,8 @@ internal Solution WithFrozenSourceGeneratedDocuments(ImmutableArray<(SourceGener
         => WithCompilationState(_compilationState.WithFrozenSourceGeneratedDocuments(documents));
 
     /// <inheritdoc cref="SolutionCompilationState.UpdateSpecificSourceGeneratorExecutionVersions"/>
-    internal Solution UpdateSpecificSourceGeneratorExecutionVersions(SourceGeneratorExecutionVersionMap sourceGeneratorExecutionVersionMap, CancellationToken cancellationToken)
-        => WithCompilationState(_compilationState.UpdateSpecificSourceGeneratorExecutionVersions(sourceGeneratorExecutionVersionMap, cancellationToken));
+    internal Solution UpdateSpecificSourceGeneratorExecutionVersions(SourceGeneratorExecutionVersionMap sourceGeneratorExecutionVersionMap)
+        => WithCompilationState(_compilationState.UpdateSpecificSourceGeneratorExecutionVersions(sourceGeneratorExecutionVersionMap));
```

Comment:
this codepath never actually made use of the cancellationtoken.  internally cleaned things up to not need this.